integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -1433,6 +1433,10 @@ msgstr "App-Team"
 msgid "Marketing team"
 msgstr "Marketing-Team"
 
+#: cms/constants/roles.py
+msgid "Limited page manager"
+msgstr "Limited page manager:in"
```

Comment:
I spoke to @karthik-integreat and we came up with "Beobachter:in" or "Leser:in". The first one is prefered